q2_feature_table/_summarize/_vega_spec.py

```diff
@@ -0,0 +1,320 @@
+          "as": "groupbyCat",
+          "expr": "datum.metadata[category]",
+          "type": "formula"
+        }
+      ],
+      "name": "table",
+      "values": values
+    },
+    {
+      "transform": [
+        {
+          "type": "aggregate",
+          "groupby": [
+            "groupbyCat"
+          ]
+        }
+      ],
+      "name": "grouped",
+      "source": "table"
+    },
+    {
+      "transform": [
+        {
+          "expr": "datum.frequency >= SamplingDepth",
+          "type": "filter"
+        },
+        {
+          "type": "aggregate",
+          "groupby": [
+            "groupbyCat"
+          ]
+        }
+      ],
+      "name": "groupedAndRetained",
+      "source": "table"
+    }
+  ],
+  "signals": [
+    {"name": "chartHeight", "value": 400},
+    {"name": "chartOffset", "value": 20},
+    {"name": "height", "update": "chartHeight + chartOffset"},
+    {
+      "name": "tooltip",
+      "value": {},
+      "on": [
+        {
+          "events": "rect:mouseover",
+          "update": "datum"
+        },
+        {
+          "events": "rect:mouseout",
+          "update": "{}"
+        }
+      ]
+    },
+    {
+      "bind": {
+        "input": "select",
+        "options": metadata_categories,
+        "element": "#metadata-category"
+      },
+      "name": "category",
+      "value": metadata_categories[0]
+    },
+    {
+      "bind": {
+        "input": "range",
+        "element": "#sampling-depth-slider",
+        "max": max_frequency,
+        "min": 0,
+        "step": 1
+      },
+      "name": "SamplingDepth",
+      "value": 0
+    },
+    {
+      "name": "rotateLabelsCheckbox",
+      "value": False,
+      "bind": {
+        "input": "checkbox",
+        "element": "#rotate-labels"
+      }
+    },
+    {
+      "name": "rotateLabels",
+      "update": "if(rotateLabelsCheckbox, 90, 0)"
+    }
+  ],
+  "marks": [
+    {
+      "description": "Interactive rarefaction summary plot",
+      "name": "rarefactionplot",
+      "type": "group",
+      "encode": {
+        "enter": {
+          "x": {
+            "value": 0
+          },
+          "y": {
+            "signal": "chartOffset"
+          },
+          "width": {
+            "value": 200
+          },
+          "height": {
+            "signal": "chartHeight"
+          }
+        }
+      },
+      "marks": [
+        {
+          "type": "rect",
+          "from": {
+            "data": "grouped"
+          },
+          "encode": {
+            "enter": {
+              "x": {
+                "scale": "xscale",
+                "field": "groupbyCat"
+              },
+              "width": {
+                "scale": "xscale",
+                "band": 1
+              },
+              "y": {
+                "scale": "yscale",
+                "field": "count"
+              },
+              "y2": {
+                "scale": "yscale",
+                "value": 0
+              }
+            },
+            "update": {
```

Comment:
```json
"tooltip": {"signal": "{\"title\": datum.groupbyCat, \"Number of Samples Dropped\": datum.count}"}
```

If you make the x-axis labels always vertical you can add a tooltip like this, to show the detail horizontally when desired.